### ISSUE TITLE
Text field component in IE family shows X button which cannot be accessed from keyboard

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [OfficeDev/office-ui-fabric-react#1216]
### Bug Report

- __Package version(s)__: 1.7.2
- __Browser and OS versions__: IE Edge/11

#### Describe the issue:

TextField component shows x button to delete the written content, only in IE Edge/11, but it cannot be accessed by the keyboard

#### Actual behavior:
Cannot reach the X button

#### Expected behavior:
Keyboard interaction should allow me to reach to that X button

**Comment by kristoferbrown** [comment]
This X button is actually a pseudo-element that IE and Edge inject into text inputs. I'm hiding this pseudo-element in order to make the TextField component IE and Edge match the one in other browsers.

A longer term solution we may want to consider is adding a similar clear button that looks and behaves the same way across all browsers and is accessible by keyboard.

**Comment by micahgodbolt** [comment]
I wonder if this should be considered an accessibility bug in IE/Edge. Where are their bug trackers?

**Comment by keepWalking** [comment]
hi, guys, I found this bug happened again in IE/Edge browser, I forked the TextField codepen: https://codepen.io/keepwalking-the-animator/full/XPdNpW/, open it using different browsers, and found "X" button still show up in IE/Edge, don't know whether this is a regression bug or not. 

**Chrome**: [no 'X']
![image](https://user-images.githubusercontent.com/4075786/44646849-e9963700-aa0e-11e8-9087-adf31941e8b5.png)

**IE11**:
![image](https://user-images.githubusercontent.com/4075786/44646814-c5d2f100-aa0e-11e8-883f-c06cf4dc6441.png)

**Edge**:
![image](https://user-images.githubusercontent.com/4075786/44646743-873d3680-aa0e-11e8-97d1-62b2cf9d1b48.png)

**Comment by micahgodbolt** [comment]
Seems this bug cropped back in again during the conversion to css-in-js….fix coming.
